tests/experimental/pydantic/test_fields.py

```diff
@@ -131,9 +132,15 @@ class UserType:
         ...
 
     assert UserType.__strawberry_definition__.fields[0].name == "friends"
+
+    if sys.version_info >= (3, 9):
+        expected_annotation = list[str]
+    else:
+        expected_annotation = List[str]
+
     assert (
```

Comment:
Peculiar that this is necessary now

Reply:
@patrick I figured it out, previously the test-case was using `StrawberryAnnotation.annotation` which is now calling `eval_type` which has some code in there calling `get_origin` which on Python >= 3.9 becomes `list` instead of `typing.List`.

I'll make a separate PR to address this for `eval_type`. For now I've changed the test to use the `raw_annotation`